src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -695,6 +695,21 @@ export const SPECIAL_CURRENCY_INFO: {
       reference: '137'
     }
   },
+  pulsechain: {
+    initWalletName: lstrings.string_first_pulsechain_wallet_name,
+    chainCode: 'PLS',
+    dummyPublicAddress: '0x0d73358506663d484945ba85d0cd435ad610b0a0',
+    allowZeroTx: true,
+    displayBuyCrypto: true,
```

Comment:
See if any of our partners supports PLS purchases and update their maps if they do. If they don't, disable this option.